spec/ttml3.xml

```diff
@@ -2537,6 +2581,17 @@ of <loc href="#terms-document-instance">document instance</loc> validity.</p>
 <p>A conforming <loc href="#conformance-generic-processor">Generic Processor</loc> is required to support the ingestion and processing
 of a <loc href="#terms-timed-text-content-document">timed text content document</loc>.</p>
 </note>
+<note role="clarification">
+<p>A <loc href="#terms-timed-text-content-document">timed text content document</loc> is not prohibited from using
+element or attribute vocabulary items defined by a <loc href="#terms-private-module">private module</loc>.
+In such case, a <loc href="#terms-content-processor">content processor</loc>
+that supports the <loc href="#terms-private-module">private module</loc> may employ one or more schemas associated with the
+<loc href="#terms-private-module">private module</loc> in order to augment a validity assessment that would normally be performed using only schemas
+associated with publicly registered modules. However, a <loc href="#terms-content-processor">content processor</loc>
+that does not support the <loc href="#terms-private-module">private module</loc> will prune unknown element or attribute vocabulary items
+according to the validity assessment rules described above, and, therefore, not report any potential invalidity that exists due to the
+use of such vocabulary items.</p>
```

Comment:
I like this clarification.

Reply:
I am adding a definition for [public module] which further clarifies between the two types of external modules (registered or not).